conn_test.go

```diff
@@ -374,6 +377,18 @@ func TestMultipleResults(t *testing.T) {
 	if val != 5 {
 		t.Fatalf("expected 5, but found %d", val)
 	}
+
+	rows, err := db.Query("CREATE TEMP TABLE temp (a int); SELECT 6;")
+	if err != nil {
+		t.Fatal(err)
+	}
+	rows.Close()
```

Comment:
Verify that the `rows` object is empty before closing it. There should also be a test where CommandComplete carries a row count and not just an empty response.